SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -613,6 +595,36 @@ bool FSpatialGDKEditorToolbarModule::GenerateDefaultLaunchConfig(const FString&
 	return false;
 }
 
+void FSpatialGDKEditorToolbarModule::GenerateSchema(bool bFullScan)
+{
+	if (bFullScan || SpatialGDKEditorInstance->FullScanRequired())
```

Comment:
Could you change the notification text if we detect no schema directory. Maybe something like "Initial Schema Generation".